File: KZP/lab3/src/ki304/rybka/lab3/PowerSupply.java
/**
 * 
 */
package ki304.rybka.lab3;

/**
 * Class PowerSupply implements power supply
 * @author Severyn Rybka
 * @version 1.0
 * @since version 1.0
 */
public class PowerSupply 
{
	private boolean isOn = false;
	/**
	 * Constructor of PowerSupply
	 */
	public PowerSupply() 
	{
		isOn = false;
	}
	/**
	 * Turn on
	 * @param on Is on
	 */
	public void turnOn(boolean on)
	{
		isOn = on;
	}
	/**
	 * If is turned on
	 * @return Is turned on
	 */
	public boolean isOn()
	{
		return isOn;
	}
}

File: KZP/lab9/power_supply.py
"""
PowerSupply
"""

class PowerSupply:
    def __init__(self):
        """ Constructor """
        self.__isOn = False


    def turnOn(self, on):
        """ Turn on """
        self.__isOn = on


    def isOn(self):
        """ If it's turned on """
        return self.__isOn
    
